packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -795,6 +795,10 @@ export class ZWaveNode
 		}
 	}
 
+	public get isFirmwareUpdateInProgress(): boolean {
```

Comment:
Should this be a getter/property or a method? Feels more like a method at first glance.